presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkQueryExecutionFactory.java

```diff
@@ -157,6 +157,8 @@
 {
     private static final Logger log = Logger.get(PrestoSparkQueryExecutionFactory.class);
     public static final String PRESTO_QUERY_ID_CONFIG = "presto_query_id";
+    public static final String PRESTO_SPARK_SHUFFLE_STATS_COLLECTOR = "shuffleStatsCollector";
```

Comment:
Do we need shuffleStatsCollector? What do we use it for? 

If we don't need it, let's remove (in a separate PR).